### PR TITLE
[doc]: Added information on developing the app

### PR DESCRIPTION
**Pull request by ericjiang97**
Added information on how to setup, run and build the app.

## Changes

- added documentation on installing flutter
- how to setup development environment
- documentation for installing dependencies
- documentation for running the app
- documentation for building the app

### CODE REVIEW

**Comment by lukesleeman** [comment]
@ericjiang97 wondering if this needs any updates based on all the problems Dylan ran into?

**Review by adamkoch — APPROVED** [review]

**Comment by ericjiang97** [comment]
@lukesleeman I reckon we can keep that to a separate doc or wiki?

**Comment by lukesleeman** [comment]
Ok, that sounds like a good idea.  Once a few flutter devs give this the ☑️, particularly Mike and Maks feel free to merge

Luke

File: README.md [hunk 1]
```diff
@@ -32,42 +31,132 @@ Everything is co-ordinated through github issues.  The current development proce
...
+flutter run
+```
+
+Otherwise if you have setup [Android Studio](https://developer.android.com/studio) and/or the Android Software Development Kit (SDK) you can run it over a virtual Android emulator using the Android Debug Bridge (ADB). Just simply run:
+
+```
+flutter run adb
+```
+
+##### Releasing
```
Comment (maks) [inline README.md]: Please remove docs about releasing, that process  is only and can only be done on CI

File: README.md [hunk 2]
```diff
@@ -32,42 +31,132 @@ Everything is co-ordinated through github issues.  The current development proce
...
+   ```
 
-*TODO:  Flutter specific information*
+   Windows:
 
-For help getting started with Flutter, view googles
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+   ```sh
+   keytool -list -v -alias key -keystore c:/Users/USER_NAME/key.jks
```
Comment (maks) [inline README.md]: Also no need for any docs about keystore and signing. Well have a private doc for admins about how this is setup in CI

**Review by maks — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ericjiang97) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (ericjiang97) [reply README.md]: Done as part of the removal of the releasing section!

**Review by domesticmouse — APPROVED** [review]

**Comment by ericjiang97** [comment]
@maks Can you please re-review

File: README.md [hunk 3]
```diff
@@ -32,42 +31,89 @@ Everything is co-ordinated through github issues.  The current development proce
...
+Currently the project is being primarily co-ordinated via the GDG Melbourne slack group. You can join GDG Melbourne's slack via [this invite](http://bit.ly/join_gdgslack). DM `@luke.sleeman` on GDG Melbourne slack to be invited to the channel being used for co-ordination.
 
 ### Pull Request Process
 
-* Develop your work in a branch.  Branch names should be prefixed with `feature`, `tech`, or `bugfix` depending on what is being done in them.  eg:  `feature\update-readme`
-* Open a pull request into master.  Be sure to include a detailed description in your pull request.  Other contributors will comment on your code or approve it
-* Please be sure to note the build status of your PR reported by the Codemagic CI
-* Once your pull request has been approved, please merge into master.
+- Develop your work in a branch. Branch names should be prefixed with `feature`, `tech`, or `bugfix` depending on what is being done in them. eg: `feature\update-readme`
```
Comment (maks) [inline README.md:113]: Sorry nitpick but it's usually / for prefix git branch name

**Review by maks — APPROVED** [review]
Just one small nit then LGTM

**Comment by rockgecko-development** [comment]
Hi @maks , re exporting your own debug cert to firebase:

> Copy either the SHA1 or the SHA256 certificate fingerprint onto the Android App on [Firebase Console]()

Is this right? It is much more convenient to include a _dev_ cert within the repo, and referenced in build.gradle for debug builds. That way if in the future Google Maps were added for example, devs won't need to go to cloud console to add their certs during dev.
Plus it's no risk to prod, because the Play Store would only allow publishing with the prod key, which can still remain safe with project owners.

The only risk I guess is someone could use this dev key to sign their own apps for Google Maps access. This could be mitigated by having a `keystore.properties` file containing the password, that is gitignored and distributed in a side channel.
This is how I have done other project in the past, but they haven't been open source.

**Comment by maks** [comment]
@rockgecko-development yes I asked for all the key and signing related content to be removed, its not  relevant for contributors, only project admins

**Comment by lukesleeman** [comment]
Looks like this has 3 approvals?  Is it good to merge?  Can somebody do that?

**Comment by ericjiang97** [comment]
Resolved #26

**Pull request merged**